src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateRequest.java

```diff
@@ -1,31 +1,44 @@
 package uk.gov.pay.directdebit.mandate.api;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.Map;
+import org.hibernate.validator.constraints.Length;
 
+import javax.validation.constraints.NotNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateMandateRequest {
 
+    @NotNull(message = "Field [return_url] cannot be null")
+    @Length(min = 1, max = 255, message = "Field [return_url] must have a size between 1 and 255")
```

Comment:
FYI you can use `{min}` and `{max}` (and any other attribute on the annotation) as a template within the message String. One benefit would be that the text in the returned message cannot get out of sync with the value used for actual validation.